src/Polly.Shared/RetrySyntaxAsync.cs

```diff
@@ -135,9 +228,24 @@ public static Policy WaitAndRetryAsync(this PolicyBuilder policyBuilder, IEnumer
         /// <returns>The policy instance.</returns>
         public static Policy WaitAndRetryAsync(this PolicyBuilder policyBuilder, int retryCount, Func<int, TimeSpan> sleepDurationProvider)
         {
+            return WaitAndRetryAsync(policyBuilder, retryCount, false, sleepDurationProvider);
+        }
+
+        /// <summary>
+        ///     Builds a <see cref="Policy"/> that will wait and retry <paramref name="retryCount"/> times.
+        ///     On each retry, the duration to wait is calculated by calling <paramref name="sleepDurationProvider"/> with
+        ///     the current retry attempt allowing an exponentially increasing wait time (exponential backoff).
+        /// </summary>
+        /// <param name="policyBuilder">The policy builder.</param>
+        /// <param name="retryCount">The retry count.</param>
+        /// <param name="continueOnCapturedContext">Whether to continue on a captured synchronization context.</param>
+        /// <param name="sleepDurationProvider">The function that provides the duration to wait for for a particular retry attempt.</param>
+        /// <returns>The policy instance.</returns>
+        public static Policy WaitAndRetryAsync(this PolicyBuilder policyBuilder, int retryCount, bool continueOnCapturedContext, Func<int, TimeSpan> sleepDurationProvider)
```

Comment:
Per previous discussion, move `bool continueOnCapturedContext` to after `sleepDurations`